docs/contributing.md

```diff
@@ -15,13 +15,14 @@ $ cd npm-package && yarn && cd ..
 If you want to debug the [NPM package](../npm-package), just run `npm link <the package path>` on your React Native project.
 
 ### Run on development mode
+_Please ensure the `React Native Debugger` production / distribution app is closed._
 
 ```bash
 $ yarn dev:webpack  # Then open the another terminal tab
 $ yarn dev:electron
 ```
-
-Please ensure the `React Native Debugger` production app is closed.
+1. From here, you can open a react-native project with remote debugging enabled. 
+1. To see the development build of the react-native-debugger, do x,y,z
```

Comment:
Running both `yarn` commands 
<a href="https://cl.ly/9d7fe5cf8eb5" target="_blank"><img src="https://d1wuojemv4s7aw.cloudfront.net/items/130o3N3z241M1T2n2c0C/%5Beca227aff0062da51ccb185bba101a08%5D_running+both+yarn+commands.png" style="display: block;height: auto;width: 100%;"/></a>

Reply:
It looks some build issue, you should check if [electron-rebuild](https://github.com/electron/electron-rebuild) is work as expected at installation. After #302 the problem will be solved.